src/cf-core/src/cf-base.less

```diff
@@ -58,66 +63,74 @@
 //
 
 body {
-    .u-webfont-regular();
-
     color: @text;
+    font-family: 'Avenir Next', Arial, sans-serif;
     font-size: unit( @base-font-size-px / 16 * 100, % );
     line-height: @base-line-height;
 }
 
+button,
+input,
+select,
+textarea {
+    // Must set these explicitly to override Normalize.css's provided default:
```

Comment:
trivial, could be `normalize.css` to match the filename.

Reply:
Well, it's also the product name, and the [website](https://necolas.github.io/normalize.css/) seems to prefer capitalizing it.

Reply:
Just thinking if we searched the codebase for references of normalize.css, this would show up along with references to the file, but I guess most searches are probably done with case-insensitivity.